core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go

```diff
@@ -99,14 +103,15 @@ type coordinator struct {
 // New creates a new CoordinatorInterface implementor.
 func New(
 	lggr logger.Logger,
+	beaconAddress common.Address,
 	coordinatorAddress common.Address,
 	dkgAddress common.Address,
 	client evmclient.Client,
 	lookbackBlocks int64,
 	logPoller logpoller.LogPoller,
 	finalityDepth uint32,
 ) (ocr2vrftypes.CoordinatorInterface, error) {
-	coordinatorContract, err := vrf_wrapper.NewVRFBeaconCoordinator(coordinatorAddress, client)
+	coordinatorContract, err := newRouter(lggr, beaconAddress, coordinatorAddress, client)
 	if err != nil {
 		return nil, errors.Wrap(err, "coordinator wrapper creation")
```

Comment:
Nit: variable name should probably change (e.g `router`) and error message too (e.g `vrf router creation`).

Reply:
created a new PR for this change: https://github.com/smartcontractkit/chainlink/pull/7536